node/src/components/consensus.rs

```diff
@@ -103,14 +105,23 @@ pub enum Event<I> {
         delay: Duration,
     },
     /// Event raised when a new era should be created: once we get the set of validators, the
-    /// booking block hash and the seed from the key block
+    /// booking block hash and the seed from the key block.
     CreateNewEra {
         /// The header of the switch block
         block_header: Box<BlockHeader>,
         /// Ok(block_hash) if the booking block was found, Err(height) if not
         booking_block_hash: Result<BlockHash, u64>,
     },
-    /// An event instructing us to shutdown if the latest era received no votes
+    /// Event raised upon initialization, when a number of eras have to be instantiated at once.
+    InitializeEras {
+        switch_blocks: HashMap<EraId, BlockHeader>,
+        current_era: EraId,
+        validators: BTreeMap<PublicKey, U512>,
+        genesis_state_root_hash: Digest,
+        timestamp: Timestamp,
+        genesis_start_time: Timestamp,
```

Comment:
This is another thing I think the era supervisor needs to get from the chainspec...